src/framework/renderer/js/RendererUtilities.js

```diff
@@ -99,12 +104,12 @@ fluid_2_0 = fluid_2_0 || {};
         return that;
     };
 
-    fluid.defaults("fluid.commonRendererComponent", {
-        gradeNames: [],
+    fluid.defaults("fluid.rendererComponent", {
+        gradeNames: ["fluid.viewComponent"],
         initFunction: "fluid.initRendererComponent",
         mergePolicy: {
             "rendererOptions.idMap": "nomerge",
-            "rendererOptions.model": "preserve",
+            //"rendererOptions.model": "preserve",
```

Comment:
This reminded me that there is still a perplexing piece of rubbish to deal with in FluidIoC.js expandCompactRec line 1831, which is the sole remaining reference to the now abolished mergePolicy of "preserve". Dead and unlamented.